### PR TITLE
Fix issues #34, #33, run PEP8

### PR DESCRIPTION
**Pull request by fernandezcuesta**
Hi, I made some minor changes to allow:
- Not entering any username, as last resort it takes local loginname
- Not entering (directly) any authentication method, if present in `~/.ssh/config` it will be used as last resort.
- `paramiko.transport` inherits handlers of loggers passed to `SSHTunnelForwarder`
- some minor changes/typos as per pep8

### CODE REVIEW

File: sshtunnel.py [hunk 1]
```diff
@@ -211,11 +227,13 @@ def get_connection_id():
 
 
 class BaseSSHTunnelForwarderError(Exception):
+
```
Comment (pahaz) [inline sshtunnel.py]: rm space

File: sshtunnel.py [hunk 2]
```diff
@@ -211,11 +227,13 @@ def get_connection_id():
...
     """ Base exception for Tunnel forwarder errors """
     pass
 
 
 class HandlerSSHTunnelForwarderError(BaseSSHTunnelForwarderError):
+
     """ Handler exception for Tunnel forwarder errors """
```
Comment (pahaz) [inline sshtunnel.py]: rm \n

File: sshtunnel.py [hunk 3]
```diff
@@ -228,6 +246,7 @@ class HandlerSSHTunnelForwarderError(BaseSSHTunnelForwarderError):
 
 
 class _ForwardHandler(SocketServer.BaseRequestHandler):
+
```
Comment (pahaz) [inline sshtunnel.py]: rm \n

File: sshtunnel.py [hunk 4]
```diff
@@ -289,13 +308,12 @@ def handle(self):
 
 
 class _ForwardServer(SocketServer.TCPServer):  # Not Threading
+
```
Comment (pahaz) [inline sshtunnel.py]: rm \n

File: sshtunnel.py [hunk 5]
```diff
@@ -322,6 +340,7 @@ def remote_port(self):
 
 
 class _ThreadingForwardServer(SocketServer.ThreadingMixIn, _ForwardServer):
+
```
Comment (pahaz) [inline sshtunnel.py]: rm \n

File: sshtunnel.py [hunk 6]
```diff
@@ -330,6 +349,7 @@ class _ThreadingForwardServer(SocketServer.ThreadingMixIn, _ForwardServer):
 
 
 class SSHTunnelForwarder(object):
+
```
Comment (pahaz) [inline sshtunnel.py]: rm \n

File: sshtunnel.py [hunk 7]
```diff
@@ -409,6 +428,7 @@ def make_ssh_forward_handler_class(self, remote_address_):
             raise BaseSSHTunnelForwarderError(msg)
 
         class Handler(_Handler):
+
```
Comment (pahaz) [inline sshtunnel.py]: rm \n

File: sshtunnel.py [hunk 8]
```diff
@@ -58,69 +58,71 @@ def do_something(port):
...
 
-__version__ = '0.0.4.4'
+__version__ = '0.0.4.5'
 __author__ = 'pahaz'
 
 __all__ = ('SSHTunnelForwarder', 'BaseSSHTunnelForwarderError',
            'HandlerSSHTunnelForwarderError', 'open_tunnel')
 
-DEFAULT_LOGLEVEL = 'DEBUG'  # Default level for logging, if no logger passed
+DEFAULT_LOGLEVEL = 40  # Default level (ERROR) for logging, if no logger passed
```
Comment (pahaz) [inline sshtunnel.py]: magic const `logging.ERROR` more readable

File: sshtunnel.py [hunk 9]
```diff
@@ -168,27 +172,39 @@ def check_addresses(address_list):
 
 def create_logger(logger=None, loglevel=DEFAULT_LOGLEVEL):
     """
-    Attaches or creates a new logger and creates console handlers if not present
+    Attaches or creates a new logger and creates console handlers if not
+    present
     """
-    logger = logger or logging.getLogger('{}.SSHTunnelForwarder'. \
+    logger = logger or logging.getLogger('{}.SSHTunnelForwarder'.
```
Comment (pahaz) [inline sshtunnel.py]: conflict (required pull)

**Comment by fernandezcuesta** [comment]
Should be ready now as requested.

**Pull request merged**